Misc/NEWS.d/next/Documentation/2021-05-25-20-35-37.bpo-12403.8BFtwS.rst

```diff
@@ -0,0 +1 @@
+Update to documentation.
```

Comment:
Update documentation of :mod:`code` and :func:`compile` to explain how their outputs can be redirected.